.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Should `.vscode` settings be in the repository?

Reply:
Yes, that'll ensure that anyone using vscode will use node_modules version of typescript, not the vscode embedded version

Reply:
What about other editors? I thought that IDE configs should be added to `.gitignore`.

Reply:
vscode is our teams preferred editor and that's the easiest way to get proper syntax highlighting. I'm not aware of other editors that have deep typescript integration like this. Other Microsoft typescript-based projects include this file in the repo (see typescript as an example)

Reply:
I asked because usually such files are excluded from the repository. If it is internal rules just ignore my comment. Maybe need to add such config to https://github.com/Microsoft/reactxp?